src/app/view/variant/variant-view/variant-history/version-entry-layout/version-entry-layout.component.ts

```diff
@@ -35,6 +39,26 @@ export class VersionEntryLayoutComponent {
     return undefined;
   }
 
+  copyComment() {
+    const comment = this.commentField;
+    if (!comment) {
+      return;
+    }
+    navigator.clipboard.writeText(comment);
+    const msg = $localize`:@@app.view.variant.variant-view.variant-history.version-entry-layout.user-id-copied:copied ${comment}:msg:`;
+    this.logService.info(msg);
+  }
+
+  copyUserId() {
+    const userId = this.getUserId();
+    if (!userId) {
+      return;
+    }
+    navigator.clipboard.writeText(userId);
+    const msg = $localize`:@@app.view.variant.variant-view.variant-history.version-entry-layout.user-id-copied:copied ${userId}:msg:`;
+    this.logService.info(msg);
```

Comment:
Suggested change: use `await navigator.clipboard.writeText()` to only show the info message once the text has successfully been copied to the clipboard. (In some cases it can take a while or fail.)